packages/atlas/src/themesource/atlas_core/web/core/_legacy/_mxui.scss

```diff
@@ -2267,17 +2267,18 @@ Main class hierarchy:
 
     /* Client defaults and helpers */
     .mx-dataview-content,
-    .mx-scrollcontainer-wrapper:not(.mx-scrollcontainer-nested),
     .mx-tabcontainer-content,
     .mx-grid-content {
         -webkit-overflow-scrolling: touch;
     }
     html,
     body,
-    #content {
+    #content,
+    #root {
```

Comment:
This root id generated by SP?

Reply:
It is generated by modern client, for some reason it changes the id of the root element, maybe it will be adjusted later by the corresponding team. We have to deal with this new id for now.